modules/olm-channel-schema.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+[id="olm-channel-schema_{context}"]
+= olm.channel schema
+
+The `olm.channel` schema defines a channel within a package, the bundle entries that are members of the channel, and the upgrade edges for those bundles.
+
+A bundle can included as an entry in multiple `olm.channel` blobs, but it can have only one entry per channel.
+
+It is valid for an entry's replaces value to reference another bundle name that cannot be found in this catalog or another catalog. However, all other channel invariants must hold true, such as a channel not having multiple heads.
```

Comment:
"all other channel invariants"... 
do we have a ref for these?  Should we?

Reply:
Leaving unresolved for further discussion + a follow-up PR.